releases/latest/full/Dockerfile.ubi.openjdk21-new

```diff
@@ -0,0 +1,32 @@
+# TODO 
```

Comment:
It should be fine to override the existing file: releases/latest/full/Dockerfile.ubi.openjdk21